assistantv1/assistant_v1_integration_test.go

```diff
@@ -3,7 +3,7 @@
 package assistantv1_test
 
 /**
- * (C) Copyright IBM Corp. 2018, 2019.
+ * (C) Copyright IBM Corp. 2018, 2020.
```

Comment:
Did you run lint copyright dates?

Reply:
I ran it pre and post changes, but it looks like it missed a couple of these files